yarn.lock

```diff
@@ -17751,9 +17751,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema#bah-contact-info":
-  version "15.0.1"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema#906698fbcf0b485b1c8850f05a5cd6f6e5999917"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema#58094bd2e593ed11b3e0a2bd18175582560bff6d":
```

Comment:
I would probably back this change out, updating this package tends to cause problems in other apps if you don't know the specific changes that are coming in.